index.html

```diff
@@ -7,54 +7,64 @@
-                <li>I have no programming skills, but I have a great desire to become a competent specialist in this profession.</br> 
-                    Personal skills that I have and, in my opinion, will help me in further learning:</li>
-                    <ul>
-                        <li>easily teachable;</li>
-                        <li>responsibility;</li>
-                        <li>seIf-improvement;</li>
-                    </ul>
-                <li><code>I dont have expirience in JS.</code></li>
-                <li>I don't have any work experience in IT sphere. My last place work:</li>
-                    <ul>
-                        <li>Middle school No 9, Brest, Belarus. Position: Educational psychologist.</br>
-                            Duties and responsibilites:</li>
-                            <ul>
-                                <li>Psychological remidial activities with students;</li>
-                                <li>Prevention and diagnostics of psyhological and mental
-                                    health problems among students;</li>
-                                <li>Documentation menedgment;</li>
-                            </ul>
-                    </ul>
-                <li>Education:</li>
-                    <ul>
-                        <li>Bachelor's Degree in Social and Medical Psychology</br>
-                            2010-2016</br>
-                            Brest State Uneversity, Faculty of Social Pedagogy (Brest,Belarus).</li>
-                        <li>Master of Psychology Candidate Concetration: Social Psychology</br>
-                            Master's thesis: <em>Psychological preparedness of high-school students for volunteering activites.</em></br> 
-                            2017-2019</br>
-                            Brest State Uneversity, Faculty of Social Pedagogy (Brest,Belarus).</li>
-                    </ul>
-                <li>English level - <b>A2-Elementary</b>.</li>
-            </ol>        
-        </section>
+    <header class="header">
+        <div class="personal-information">
+            <h1>KUZMIN DZMITRY</h1>
+            <div class="objective">
+                <h3>Objective: Junior Front-End Developer</h3>
+            </div>
+            <div class="contact">
+                <img src="/images/icon-phone.svg" alt="phone">+372(33)323-17-35
+            </div>
+            <div class="contact">
+                 <a href="mailto:dimakuzmin9@gmail.com">
+                     <img src="/images/icon-email.svg" alt="email" >dimakuzmin9@gmail.com
+                </a>
+            </div>
+        </div>
+        <div class="avatar-container">
+                <img class="avatar" src="/images/photo-dima.jpg" alt="Kuzmin Dzmitry photo">
+        </div>
+    </header>
+    <div class="introduction">
+        <p>After working as a school psychologist, I decided that I didn't want to have anything with our political system.
```

Comment:
class="introduction" can be applied to `<p>` and `<div>` is not needed then.

Reply:
fixed